modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/configuration/RedirectURLSettingsImpl.java

```diff
@@ -5,46 +5,141 @@
 
 package com.liferay.redirect.internal.configuration;
 
+import com.liferay.portal.configuration.metatype.bnd.util.ConfigurableUtil;
+import com.liferay.portal.kernel.model.CompanyConstants;
 import com.liferay.portal.kernel.redirect.RedirectURLSettings;
-import com.liferay.redirect.internal.configuration.admin.service.RedirectURLManagedServiceFactory;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.HashMapDictionaryBuilder;
 
+import java.util.Dictionary;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.Constants;
+import org.osgi.framework.ServiceRegistration;
+import org.osgi.service.cm.ConfigurationException;
+import org.osgi.service.cm.ManagedServiceFactory;
+import org.osgi.service.component.annotations.Activate;
 import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Modified;
 
 /**
  * @author Drew Brokke
  */
-@Component(service = RedirectURLSettings.class)
+@Component(
+	configurationPid = "com.liferay.redirect.internal.configuration.RedirectURLConfiguration",
```

Comment:
Hey @RenanUmV, Why did you copy the `RedirectURLManagedServiceFactory` logic to here? Please check if it is needed. Thx

Reply:
I forgot to delete the RedirectURLManagedServiceFactory class